source/developers/projects/engine/api/access-to-services.rst

```diff
@@ -0,0 +1,41 @@
+:is-up-to-date: True
+
+:orphan:
+
+.. _access-to-services:
+
+==================
+Access to Services
+==================
+
+When developing templates or scripts only a small list of services are available to use, you can expose other
+services with the following steps.
+
```

Comment:
Maybe break up the sentence into two sentences?

When developing templates or scripts only a small list of services are available to  use.  You can expose other services with the following steps.